src/components/Nav/Nav.scss

```
@@ -0,0 +1,53 @@
+@import '/src/styles/mixin.scss';
+@import '/src/styles/variables.scss';
+
+.nav {
+  @include flexSort(space-between, center);
+  background-color: white;
+  padding: 0px 15%;
+  height: 80px;
+  @include initBorderStyle(bottom);
```

```suggestion
  @include flexSort(space-between, center);
    @include initBorderStyle(bottom);
  background-color: white;
  padding: 0px 15%;
  height: 80px;
```

같은 include 끼리 모아 놓는 게 좋습니다~

---

넵 main브랜치로 푸시할 상황이 생기면 바로 반영하겠습니다.